### PR TITLE
AclField factory function + graphene.Enum registry support

### PR DESCRIPTION
**Pull request by nikordaris**

### CODE REVIEW

File: graphene_acl/fields.py [hunk 1]
```diff
@@ -16,8 +16,9 @@ def __init__(
         super().__init__(*args, **kwargs)
 
     def get_resolver(self, parent_resolver):
+        super_resolver = super().get_resolver(parent_resolver)
```
Comment (timthesinner) [inline graphene_acl/fields.py]: Should be `super(AclField, self)`

File: graphene_acl/fields.py [hunk 2]
```diff
@@ -39,3 +40,11 @@ def decorator(func):
             return func
 
         return decorator
+
+
+def acl_field_type(name, Field):
```
Comment (timthesinner) [inline graphene_acl/fields.py]: Should we change `Field` to `WrappedField`?  May be confused with imports?

File: graphene_acl/fields.py [hunk 3]
```diff
@@ -39,3 +40,11 @@ def decorator(func):
...
+    def get_resolver(self, parent_type):
```
Comment (timthesinner) [inline graphene_acl/fields.py]: Use `parent_resolver` instead of `parent_type`

File: graphene_acl/routing.py [hunk 4]
```diff
@@ -16,18 +17,22 @@ def __init__(self, classifier=None):
 
     def register(self, handler, key=None):
         if key:
-            self._handlers[key] = handler
+            if isinstance(key, Enum):
```
Comment (timthesinner) [inline graphene_acl/routing.py]: Do we also need to handle `graphene.Enum` or is that type captured by stdlib `Enum`?

Comment (nikordaris) [reply graphene_acl/routing.py]: ya its captured by Enum. graphene.Enum actually doesn't even work....

**Pull request merged**